.github/workflows/add-to-project.yml

```diff
@@ -0,0 +1,35 @@
+name: Add To Project
+on:
+  issues:
+    types: [opened]
+  workflow_call:
+    secrets:
+      ADD_TO_PROJECT_PAT:
+        required: true
+
+env:
+  DESIGN_SYSTEM_PROJECT_URL: https://github.com/orgs/carbon-design-system/projects/39
+  PROPOSALS_PROJECT_URL: https://github.com/orgs/carbon-design-system/projects/51
+  LABEL_ENHANCEMENT: 'type: enhancement 💡'
+
+jobs:
+  add-to-proposals-project:
+    name: Add issue with enhancement label to the Proposals project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.4.0
+        with:
+          labeled: ${{ env.LABEL_ENHANCEMENT }}
+          project-url: ${{ env.PROPOSALS_PROJECT_URL }}
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
@sstrubberg Based on our conversations yesterday, I just added this here that will add issues with the https://github.com/carbon-design-system/carbon/labels/type%3A%20enhancement%20%F0%9F%92%A1 label to the Proposals project. These will no longer be added to the Design System project by default. Let me know what you think!